rootfs/etc/services.d/acarsdec/run

```diff
@@ -7,7 +7,7 @@
 if [ -n "${ENABLE_ACARS}" ]; then
 
 	ACARS_BIN="/usr/local/bin/acarsdec"
-	ACARS_CMD=("-g" "$GAIN" "-i" "$STATION_ID_ACARS")
+	ACARS_CMD=("-g" "$GAIN_ACARS" "-i" "$STATION_ID_ACARS")
```

Comment:
Hi @fredclausen,

To allow backwards compatibility with `GAIN`, I suggest we do something like:

```bash
if [ -n "$GAIN" ]; then
    GAIN_ACARS="$GAIN"
fi
```

What do you think?

Reply:
Excellent call...done